yosys/miter_and_tempinduct_large_N.yosys

```diff
@@ -0,0 +1,5 @@
+hierarchy
+proc
+clk2fflogic
+miter -equiv -flatten -ignore_gold_x -make_outputs -make_outcmp gold gate miter
+sat -prove trigger 0 -verify -tempinduct -tempinduct-skip 1 -initsteps 32 -stepsize 16 miter
```

Comment:
Do the `-initsteps` and `-stepsize` values matter at all?

Reply:
Yes, I'll add a comment